common/projectconfig/projectconfig.go

```diff
@@ -92,6 +97,21 @@ func loadFile(path string) (Config, error) {
 	return config, nil
 }
 
+func CreateAndSave(config Config) error {
+	path := GetDefaultConfigPath()
+	_, err := os.Stat(path)
+	// Only create a new file if there isn't one already defined at this location
+	if err != nil && errors.Is(err, os.ErrNotExist) {
+		if err = os.WriteFile(path, []byte{}, 0600); err != nil {
```

Comment:
As per above, I think this function is redundant, but just FTR it's always better to write to a temp file and rename it, rather than overwrite a file directly, because if the write fails halfway through it will result in a corrupted config. With the rename temp file approach, the rename is atomic, so the file is either old or new, never "half written".

Reply:
Ah, that's very good to know. Thanks!